src/Geopilot.Api/ContextExtensions.cs

```diff
@@ -97,18 +97,29 @@ public static void SeedOrganisations(this Context context)
         context.SaveChanges();
     }
 
-    public static Geometry GetExtent(this Address address)
+    public static Geometry GetExtent()
     {
-        var longitude = address.Longitude();
-        var latitude = address.Latitude();
+        // Reference extent for Switzerland
+        var longMinRef = 7.536621;
+        var latMinRef = 46.521076;
+        var longMaxRef = 9.398804;
+        var latMaxRef = 47.476376;
```

Comment:
PP: In Konstanten auslagern und entsprechend benennen, zB `longMinRefChExtent`, `longMinRefCh`, oder was Ähnliches